.github/workflows/no-text.yml

```diff
@@ -0,0 +1,32 @@
+name: no-text
+on:
+  - push
+  - pull_request
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: ['ubuntu-latest']
+        ghc: ['latest']
+      fail-fast: false
+    steps:
+    - uses: actions/checkout@v2
+    - uses: haskell/actions/setup@v1
+      id: setup-haskell-cabal
+      with:
+        ghc-version: ${{ matrix.ghc }}
+    - name: Update cabal package database
+      run: cabal update
+    - uses: actions/cache@v2
+      name: Cache cabal stuff
+      with:
+        path: ${{ steps.setup-haskell-cabal.outputs.cabal-store }}
+        key: ${{ runner.os }}-${{ matrix.ghc }}
+    - name: Build
+      run: cabal build -f-text prettyprinter:prettyprinter
```

Comment:
Running the tests would be nice, but that's also something that we can try once we feel a stronger need.

Reply:
Seems I found a way to wire in tests without too much churn, pushed.